apps/frontend/components/DisasterIcon/DisasterIcon.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  faBiohazard,
+  faBomb,
+  faBug,
+  faBuildingCircleXmark,
+  faCarBurst,
+  faCloudBolt,
+  faEllipsis,
+  faFireFlameSimple,
+  faHillRockslide,
+  faHouseFloodWater,
+  faHurricane,
+  faPersonDrowning,
+  faSunPlantWilt,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { DisasterMapping, IncidentMapping } from '@wfp-dmp/interfaces';
+import { useMemo } from 'react';
+import { useIntl } from 'react-intl';
+
+// eslint-disable-next-line complexity
+const getIconDefinition = (disTyp: string) => {
+  switch (disTyp) {
+    case DisasterMapping.flood:
```

Comment:
Just a note for the future:

This is another place where having a disType to `{object}` mapping would have been useful.
All of this could have live in the same mapping returning `{id, icon}` and other info we may need.